planner/src/main/java/com/dask/sql/schema/DaskTable.java

```diff
@@ -23,16 +25,24 @@
  *
  * Basically just a list of columns, each column being a column name and a type.
  */
-public class DaskTable implements ProjectableFilterableTable {
+public class DaskTable implements TranslatableTable {
 	// List of columns (name, column type)
 	private final ArrayList<Pair<String, SqlTypeName>> tableColumns;
 	// Name of this table
 	private final String name;
+	// Any statistics information we have
+	private final DaskStatistics statistics;
 
-	/// Construct a new table with the given name
-	public DaskTable(final String name) {
+	/// Construct a new table with the given name and estimated row count
+	public DaskTable(final String name, final Double rowCount) {
 		this.name = name;
 		this.tableColumns = new ArrayList<Pair<String, SqlTypeName>>();
+		this.statistics = new DaskStatistics(rowCount);
```

Comment:
I don't believe it would revert it back to _exactly_ like it was pre-cbo however without the statistics the more fine grained CBO improvements would not be available. Especially optimizations around data types, sizes, or number of rows.